### ISSUE TITLE
dividerColor does not work on Android

### ISSUE THREAD

**Issue by tapz** [henninghall/react-native-date-picker#787]
**Describe the bug**
Dividers stay black even if dividerColor='red' (or any hex value).

**Expected behavior**
Divider colors changes to red.

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
<DatePicker
  style={styles.datePicker}
  date={new Date(2100, 0, 1, 0, 0, 0, 0)}
  locale={'en-FI'}
  mode='time'
  onDateChange={props.onTimeChanged}
  dividerColor='red'
  theme='light'
  minuteInterval={15}
  is24hourSource='locale'
 />

const styles = StyleSheet.create({
  datePicker: {
    alignSelf: 'center',
    width: 120
  }
});
```

**Smartphone (please complete the following information):**
 - OS: Android 14
 - React Native version 0.73.6
 - react-native-date-picker version 5.0.0

**Comment by efstathiosntonas** [comment]
same here on version `5.0.0`

**Comment by henninghall** [comment]
Thanks for reporting this. It is fixed in version 5.0.1
